timescaledb/overview/release-notes/index.md

```diff
@@ -16,8 +16,10 @@ community][timescale-slack].
 
 For our next releases, we plan to add:
 
-- Continuous aggregates with compression
-- `time_bucket_ng` suport for N months and Timezones on continuous aggregates
+- Performance improvements for:
+  - Inserts and common queries on Multi-node 
```

Comment:
Is it for 2.7? We already have notable improvements in 2.6, maybe makes sense to mention them as well.

Reply:
We do have a list of them for 2.7 planned, yes.

Reply:
So why are the performance improvements of 2.6 over 2.5 not mentioned in the release notes?

Reply:
Added to the changelog https://github.com/timescale/timescaledb/pull/4084/commits/9f83a03a6ff1ecd5b781fc845640a2fedafba5c4